color-identifiers-mode.el

```diff
@@ -337,6 +337,16 @@ arguments, loops (for .. in), or for comprehensions."
                   "\\_<\\([a-zA-Z_$]\\(?:\\s_\\|\\sw\\)*\\)"
                   (nil font-lock-variable-name-face tree-sitter-hl-face:variable))))
 
+;; Python Tree-sitter
+(color-identifiers:set-declaration-scan-fn
+ 'python-ts-mode 'color-identifiers:python-get-declarations)
```

Comment:
Function `color-identifiers:python-get-declarations` is really a workaround for vanilla `python-mode` not highlighting function parameters and variable assignments in any meaningful way to be used by `color-identifers-mode`. *(or at least it didn't highlight them back when the function was implemented)*.

Does `python-ts-mode` still not implement highlighting function parameters and `foo = bar` assignments with  `font-lock-variable-name-face` or something similar?